scripts/pi-hole/js/groups-adlists.js

```diff
@@ -117,24 +117,32 @@ function initTable() {
     order: [[0, "asc"]],
     columns: [
       { data: "id", visible: false },
+      { data: null, visible: true, orderable: false, width: "15px" },
       { data: "status", searchable: false, class: "details-control" },
-      { data: "address" },
+      { data: "address", orderable: false },
```

Comment:
I know it was like not orderable before, but I don't see a reason why it should not be orderable...

```suggestion
      { data: "address" },
```

Reply:
Changed